src/main/java/codeu/controller/AdminServlet.java

```diff
@@ -12,14 +12,16 @@
 import java.util.ArrayList;
 import java.util.List;
 import org.mindrot.jbcrypt.BCrypt;
+import java.util.UUID;
+
 
 /** Servlet class responsible for the Admin page. */
 public class AdminServlet extends HttpServlet {
 
-  private UserStore UserStore;
-  private ConversationStore ConversationStore;
-  private MessageStore MessageStore;
-
+  private UserStore userStore;
+  private ConversationStore conversationStore;
+  private MessageStore messageStore;
+  private List<String> adminList = new ArrayList<>();
```

Comment:
I'm also a bit confused on why you're maintaining a list of admins since you're checking if the user is an admin in the beginning and adding him to the admin list. Why not always just check if the user is an admin by using getIsAdmin throughout. Wouldn't it be less expensive than searching through a list?